<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Tab_Init</class>
 <widget class="QWidget" name="Tab_Init">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QGroupBox" name="gb_comp">
     <property name="title">
      <string>Composition</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="edit_composition">
        <property name="toolTip">
         <string>Type the composition of the crystal here,

Examples:
ti1o2
Ti1O2
TI1O2
Ti1 O2
Ti 1 o 2
etc.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_14">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Cell composition:</string>
        </property>
        <property name="buddy">
         <cstring>edit_composition</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QTableWidget" name="table_comp">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <property name="sortingEnabled">
         <bool>true</bool>
        </property>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>9</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Symbol</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Z</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>#</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Mass</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Min. Radius</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1" rowspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Unit Cell Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Length B (Å):</string>
        </property>
        <property name="buddy">
         <cstring>spin_b_min</cstring>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Length C (Å):</string>
        </property>
        <property name="buddy">
         <cstring>spin_c_min</cstring>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Angle α (°):</string>
        </property>
        <property name="buddy">
         <cstring>spin_alpha_min</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Length A (Å):</string>
        </property>
        <property name="buddy">
         <cstring>spin_a_min</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QDoubleSpinBox" name="spin_a_min">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QDoubleSpinBox" name="spin_b_min">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QDoubleSpinBox" name="spin_c_min">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>3.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QDoubleSpinBox" name="spin_alpha_min">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>60.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="spin_a_max">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QDoubleSpinBox" name="spin_b_max">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="3" column="3">
       <widget class="QDoubleSpinBox" name="spin_c_max">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>9999.989999999999782</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>10.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="4" column="3">
       <widget class="QDoubleSpinBox" name="spin_alpha_max">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>120.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label_6">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>Minimum</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>Maximum</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Angle β (°):</string>
        </property>
        <property name="buddy">
         <cstring>spin_beta_min</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:10pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Volume (Å&lt;span style=&quot; vertical-align:super;&quot;&gt;3&lt;/span&gt;):&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
        <property name="buddy">
         <cstring>spin_vol_min</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QDoubleSpinBox" name="spin_beta_min">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>60.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QDoubleSpinBox" name="spin_gamma_min">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>60.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QDoubleSpinBox" name="spin_vol_min">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>0.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="5" column="3">
       <widget class="QDoubleSpinBox" name="spin_beta_max">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>120.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="8" column="3">
       <widget class="QDoubleSpinBox" name="spin_vol_max">
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>1000000.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>1.000000000000000</double>
        </property>
        <property name="value">
         <double>1000000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="3">
       <widget class="QDoubleSpinBox" name="spin_gamma_max">
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>180.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>120.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Angle γ (°):</string>
        </property>
        <property name="buddy">
         <cstring>spin_gamma_min</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2" rowspan="8">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
       </widget>
      </item>
      <item row="10" column="3">
       <widget class="QDoubleSpinBox" name="spin_fixedVolume">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Use this to specify a starting volume for the structure before optimizing.</string>
        </property>
        <property name="decimals">
         <number>5</number>
        </property>
        <property name="maximum">
         <double>999999.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
        <property name="value">
         <double>500.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="2">
       <widget class="QCheckBox" name="cb_fixedVolume">
        <property name="toolTip">
         <string>Use this to specify a starting volume for the structure before optimizing.</string>
        </property>
        <property name="text">
         <string>Fixed volume:</string>
        </property>
       </widget>
      </item>
      <item row="12" column="3">
       <widget class="QDoubleSpinBox" name="spin_scaleFactor">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Use this to limit the shortest interatomic distance between
atoms. This ensures that atoms aren't overlapping.

The value specified here is multiplied by each atom's covalent 
radius, and the sum of the scaled radii are used to enforce a
minimum atomic separation between atoms.

The default recommended value is 0.5. This ensures that atoms will not be unphysically close (which may prevent
certain optimization codes from working). Using a higher value
is acceptable while creating random structures in the first 
generation, but may prevent the crossover operator from being
able to create offspring that meet this restriction.</string>
        </property>
        <property name="suffix">
         <string> * radii</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="maximum">
         <double>1.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.500000000000000</double>
        </property>
       </widget>
      </item>
      <item row="14" column="3">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="12" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QCheckBox" name="cb_interatomicDistanceLimit">
          <property name="toolTip">
           <string>Use this to limit the shortest interatomic distance between atoms. This ensures that atoms aren't overlapping.</string>
          </property>
          <property name="text">
           <string>Limit interatomic distance?</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Scale factor:</string>
          </property>
          <property name="buddy">
           <cstring>spin_scaleFactor</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="13" column="0" colspan="2">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_9">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Minimum radius:</string>
          </property>
          <property name="buddy">
           <cstring>spin_minRadius</cstring>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="13" column="3">
       <widget class="QDoubleSpinBox" name="spin_minRadius">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="suffix">
         <string> Å</string>
        </property>
        <property name="decimals">
         <number>2</number>
        </property>
        <property name="singleStep">
         <double>0.050000000000000</double>
        </property>
        <property name="value">
         <double>0.250000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Crystal &amp;type:</string>
       </property>
       <property name="buddy">
        <cstring>combo_type</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_type">
       <item>
        <property name="text">
         <string>Atomistic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Molecular</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="gb_mcomp">
     <property name="title">
      <string>Molecular Composition</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTableWidget" name="table_mcomp">
        <property name="editTriggers">
         <set>QAbstractItemView::NoEditTriggers</set>
        </property>
        <property name="selectionBehavior">
         <enum>QAbstractItemView::SelectRows</enum>
        </property>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <column>
         <property name="text">
          <string>Quantity</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Filename</string>
         </property>
        </column>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>Maximum conformations:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spin_maxConf">
          <property name="toolTip">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;Maxmium number of conformations to find for each molecular component&lt;br&gt;&lt;br&gt;These are randomly selected during random structure generation and used in the &amp;quot;Reconf&amp;quot; operator.&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="push_add">
          <property name="text">
           <string>&amp;Add</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="push_remove">
          <property name="text">
           <string>&amp;Remove</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edit_composition</tabstop>
  <tabstop>spin_a_min</tabstop>
  <tabstop>spin_a_max</tabstop>
  <tabstop>spin_b_min</tabstop>
  <tabstop>spin_b_max</tabstop>
  <tabstop>spin_c_min</tabstop>
  <tabstop>spin_c_max</tabstop>
  <tabstop>spin_alpha_min</tabstop>
  <tabstop>spin_alpha_max</tabstop>
  <tabstop>spin_beta_min</tabstop>
  <tabstop>spin_beta_max</tabstop>
  <tabstop>spin_gamma_min</tabstop>
  <tabstop>spin_gamma_max</tabstop>
  <tabstop>spin_vol_min</tabstop>
  <tabstop>spin_vol_max</tabstop>
  <tabstop>cb_fixedVolume</tabstop>
  <tabstop>spin_fixedVolume</tabstop>
  <tabstop>cb_interatomicDistanceLimit</tabstop>
  <tabstop>spin_scaleFactor</tabstop>
  <tabstop>spin_minRadius</tabstop>
  <tabstop>table_comp</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>cb_interatomicDistanceLimit</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_scaleFactor</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>282</y>
    </hint>
    <hint type="destinationlabel">
     <x>908</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_fixedVolume</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_fixedVolume</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>768</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>908</x>
     <y>256</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_fixedVolume</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_vol_min</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>768</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>768</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_fixedVolume</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_10</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>768</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>616</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_fixedVolume</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_vol_max</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>768</x>
     <y>255</y>
    </hint>
    <hint type="destinationlabel">
     <x>908</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_interatomicDistanceLimit</sender>
   <signal>toggled(bool)</signal>
   <receiver>label</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>541</x>
     <y>274</y>
    </hint>
    <hint type="destinationlabel">
     <x>720</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_interatomicDistanceLimit</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_9</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>701</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cb_interatomicDistanceLimit</sender>
   <signal>toggled(bool)</signal>
   <receiver>spin_minRadius</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>511</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>846</x>
     <y>301</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
